### PR TITLE
doc: 6725 add pcap file logging variable details v1

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at
   https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at
   https://suricata.io/about/contribution-agreement/ (note: this is only required once)
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/6725

Describe changes:
- add pcap file name variables documentation
-
-

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

File: doc/userguide/configuration/suricata-yaml.rst [hunk 1]
```diff
@@ -505,6 +505,26 @@ the alert.
...
+In normal mode a pcap file "filename" is created in the default-log-dir or as
+specified by "dir".
+
+In multi mode, a file is created per thread. This will perform much better but
+will create multiple files where 'normal' would create a single file.
+
+In multi mode the filename takes a few special variables:
+  - %n representing the thread number
+  - %i representing the thread id
+  - %t representing the timestamp (secs or secs.usecs based on 'ts-format'
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:517]: Nit: missing closing parenthesis

File: doc/userguide/configuration/suricata-yaml.rst [hunk 2]
```diff
@@ -505,6 +505,26 @@ the alert.
...
+  
+  Example: filename: pcap.%n.%t
+
+.. note:: It is possible to use directories but the directories are not
+  created by Suricata. For example ``filename: pcaps/%n/log.%s`` will log into
+  the per thread directory.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:523]: Should we emphasize in the example something like `will log into the pre-existing pcap directory, in per thread folders`. Or something like that, better worded?

File: doc/userguide/configuration/suricata-yaml.rst [hunk 3]
```diff
@@ -505,6 +505,26 @@ the alert.
       mode: normal # "normal" or multi
       conditional: alerts
 
+In normal mode a pcap file "filename" is created in the default-log-dir or as
+specified by "dir".
+
+In multi mode, a file is created per thread. This will perform much better but
+will create multiple files where 'normal' would create a single file.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:512]: nit could add "at the cost of performance." at the end?

**Review by jufajardini — CHANGES_REQUESTED** [review]
Thanks for this addition!
A few mostly nit comments from my end.

**Comment by jmtaylor90** [comment]
continued in https://github.com/OISF/suricata/pull/10293

**Pull request closed without merging**